### PR TITLE
IAM account settings & users

### PR DESCRIPTION
**Pull request by osterman**
## what
* Upgrade geodesic
* Add `account-settings` module
* Add `users` module

## why
* Facilitate gitops style operations for user management

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[submodule "conf/atlantis"]
+	path = conf/atlantis
+	url = https://github.com/cloudposse/geodesic-aws-atlantis.git
+	branch = init
```
Comment (aknysh) [inline .gitmodules]: should use branch `master` (after we release `geodesic-aws-atlantis`)

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
-FROM cloudposse/terraform-root-modules:0.5.4 as terraform-root-modules
+FROM cloudposse/terraform-root-modules:0.6.0 as terraform-root-modules
 
-FROM cloudposse/geodesic:0.21.0
+FROM cloudposse/geodesic:0.37.0
```
Comment (aknysh) [inline Dockerfile]: `geodesic` already at `0.38.0`

File: conf/users/osterman.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
+module "osterman" {
+  source  = "git::https://github.com/cloudposse/terraform-aws-iam-user.git?ref=0.1.0"
+  name    = "osterman"
+  pgp_key = "keybase:osterman"
+  groups  = "${local.admin_groups}"
+}
+
+module "osterman_welcome" {
+  source   = "git::https://github.com/cloudposse/terraform-null-smtp-mail.git?ref=master"
```
Comment (aknysh) [inline conf/users/osterman.tf]: pin to a release

File: conf/users/terraform.tfvars [hunk 4]
```diff
@@ -0,0 +1,29 @@
+terragrunt = {
+  terraform {
+    source = "git::git@github.com:cloudposse/terraform-root-modules.git//aws/users?ref=iam-settings"
+    extra_arguments "smtp" {
```
Comment (aknysh) [inline conf/users/terraform.tfvars]: why the name `smtp` for TF commands?

File: .gitmodules (hunk 1 above)
Comment (osterman) [reply .gitmodules]: I think a tag... `0.1.0`

File: conf/users/terraform.tfvars (hunk 4 above)
Comment (osterman) [reply conf/users/terraform.tfvars]: Naming the "extra" arguments for what they do: cast `SMTP_` envs to terraform envs.

**Comment by osterman** [comment]
atlantis/root plan

**Comment by osterman** [comment]
atlantis/root plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis/root plan

**Comment by osterman** [comment]
atlantis/root apply

**Comment by osterman** [comment]
atlantis/root apply

**Comment by aknysh** [comment]
atlantis/root plan

**Pull request merged**